app/jobs/resolution_proofing_job.rb

```diff
@@ -84,24 +89,22 @@ def add_threatmetrix_result_to_callback_result(callback_log_data_result, threatm
     callback_log_data_result[:threatmetrix_request_id] = threatmetrix_result.transaction_id
   end
 
-  def proof_lexisnexis_ddp_with_threatmetrix_if_needed(
-    applicant_pii,
-    user_id,
-    threatmetrix_session_id
-  )
+  def proof_lexisnexis_ddp_with_threatmetrix_if_needed(opt)
```

Comment:
can we turn this into keyword args? They're much clearer about what inputs are needed

Reply:
done

Reply:
Hi sorry to be clear, I meant, "can we use the keyword arguments syntax" not just rename this to `args`. This is more of what I meant:

```ruby
  def proof_lexisnexis_ddp_with_threatmetrix_if_needed(
    applicant_pii:,
    user_id:,
    threatmetrix_session_id:,
    request_ip:,
    uuid_prefix:
  )
```

Reply:
I tried that, there's a code climate limitation, which is why I went this way. Any other thoughts?

Method proof_lexisnexis_ddp_with_threatmetrix_if_needed has 5 arguments (exceeds 4 allowed). Consider refactoring. NEW
  def proof_lexisnexis_ddp_with_threatmetrix_if_needed(applicant_pii, user, threatmetrix_session_id, request_ip, uuid_prefix)
Severity: MinorFound in [app/jobs/resolution_proofing_job.rb](https://codeclimate.com/github/18F/identity-idp/app/jobs/resolution_proofing_job.rb?from_sha=35645733&to_sha=97c9d6e2) - About 35 mins to fix